roles/kubernetes/node/templates/node-kubeconfig.yaml.j2

```diff
@@ -4,6 +4,7 @@ clusters:
 - name: local
   cluster:
     certificate-authority: {{ kube_cert_dir }}/ca.pem
+    server: http://127.0.0.1:8080
```

Comment:
There is {{ kube_apiserver_endpoint }} variable. The value will be different for k8s master and k8s node. Let's use the variable instead of hardcoding.